emscripten.py

```diff
@@ -2163,7 +2195,8 @@ def add_metadata_from_wast(metadata, wast):
       if import_type == 'memory':
         continue
       elif import_type == 'func':
-        if not import_name.startswith('invoke_'):
+        if not import_name.startswith('invoke_') and \
```

Comment:
Here looks really odd to me with the backslash continuation

Reply:
Changed to
```
        if (not import_name.startswith('invoke_') and                            
            not import_name.startswith('jsCall_'):
```